storio-common/src/main/java/com/pushtorefresh/storio2/operations/internal/FlowableOnSubscribeExecuteAsBlockingOptional.java

```diff
@@ -0,0 +1,36 @@
+package com.pushtorefresh.storio2.operations.internal;
+
+import android.support.annotation.NonNull;
+
+import com.pushtorefresh.storio2.Optional;
+import com.pushtorefresh.storio2.operations.PreparedOperation;
+
+import io.reactivex.FlowableEmitter;
+import io.reactivex.FlowableOnSubscribe;
+
+/**
+ * Required to avoid problems with ClassLoader when RxJava is not in ClassPath
+ * We can not use anonymous classes from RxJava directly in StorIO, ClassLoader won't be happy :(
+ * <p>
+ * For internal usage only!
+ */
+public final class FlowableOnSubscribeExecuteAsBlockingOptional<Result, Data> implements FlowableOnSubscribe<Optional<Result>> {
+
+    @NonNull
+    private final PreparedOperation<Result, Optional<Result>, Data> preparedOperation;
+
+    public FlowableOnSubscribeExecuteAsBlockingOptional(@NonNull PreparedOperation<Result, Optional<Result>, Data> preparedOperation) {
+        this.preparedOperation = preparedOperation;
+    }
+
+    @Override
+    public void subscribe(@NonNull FlowableEmitter<Optional<Result>> emitter) throws Exception {
+        try {
+            final Result value = preparedOperation.executeAsBlocking();
+            emitter.onNext(Optional.of(value));
+            emitter.onComplete();
```

Comment:
Are you sure that it has to complete here?

Reply:
I think yes. We use this flowable only for starting emission or if have no tables/tags to observe. 
Another flowable created by `ChangesFilter.applyForTablesAndTags` will not complete